src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -22,8 +22,8 @@
     private static final Logger logger = LogsCenter.getLogger(UiManager.class);
     private static final String ICON_APPLICATION = "/images/address_book_32.png";
 
-    private Logic logic;
-    private MainWindow mainWindow;
+    private static Logic logic;
+    private static MainWindow mainWindow;
```

Comment:
is it safe to make it static? not sure if itll affect, for example, when there are multiple UiManagers